examples/basic/index.html

```diff
@@ -153,15 +151,23 @@
 }
 
 function onResize() {
```

Comment:
The biggest most persistent issue I see is that on iOS, if your phone is already in landscape mode, the google cardboard notification will not go away. You have to switch to portrait and back before it will go away. But the WebVR polyfill always did this, it was not anything introduced by these changes.